apps/face-recognition.sh

```diff
@@ -87,10 +87,20 @@ then
     exit 1
 fi
 
+
+echo "deb https://repo.delellis.com.ar focal focal" > /etc/apt/sources.list.d/20-pdlib.list
+if ! curl -fsSL https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
+then
+    msg_box "It seems like the gpg key for the needed PHP dependency couldn't get added.
+This is most likely just a temporary downtime of the server where the key comes from.
+Please try again later!"
+    exit
+fi
+
 # Install requirements
 # https://github.com/matiasdelellis/facerecognition/wiki/Installation#ubuntu-focal
 echo "deb https://repo.delellis.com.ar focal focal" > /etc/apt/sources.list.d/20-pdlib.list
-wget -qO - https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
+curl_to_dir https://repo.delellis.com.ar repo.gpg.key . | sudo apt-key add -
```

Comment:
Why not add it under `/etc/apt/trusted.gpg.d/` ?

Reply:
Yeah why not? :)

Please change.

Reply:
Can we do the facerecognition part in https://github.com/nextcloud/vm/pull/2001?

Reply:
this improves all the scripts, and when we merge this we would change that part anyway so I think it's better to continue here.